tests/plugins/wysiwygarea/manual/detached.html

```diff
@@ -0,0 +1,37 @@
+<button id="toggleButton">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div id="editorContainer">
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+<script>
+	var editorElement = document.getElementById( 'editor' ),
+		editorContainer = document.getElementById( 'editorContainer' ),
+		editorContainerParent = editorContainer.parentNode;
+
+	var editor = CKEDITOR.replace( editorElement );
+	editor.setData( '<p>ole!</p>' );
+
+	var toggleButton = CKEDITOR.document.getById( 'toggleButton' ),
+		setButton = CKEDITOR.document.getById( 'setButton' );
+
+	setButton.on ( 'click', function() {
+		editor.setData( 'any any any' );
+	});
+
+	var editorDetached = false;
```

Comment:
Now it's clear we could have just one `var` statement for all of those variables.